initPTB.m

```diff
@@ -152,3 +103,83 @@
 
 
 end
+
+
+function initDebug(cfg)
+
+% init PTB with different options in concordance to the debug Parameters
+Screen('Preference','SkipSyncTests', 0);
+if cfg.debug
+    
+    Screen('Preference', 'SkipSyncTests', 2);
+    Screen('Preference', 'Verbosity', 0);
+    Screen('Preferences', 'SuppressAllWarnings', 2);
+    
+    fprintf('\n\n\n\n')
+    fprintf('########################################\n')
+    fprintf('##   DEBUG MODE. TIMING WILL BE OFF.  ##\n')
+    fprintf('########################################')
+    fprintf('\n\n\n\n')
+    
+end
+
+if cfg.testingTranspScreen
+    PsychDebugWindowConfiguration
+end
+
+end
+
+function initKeyboard(cfg)
+
+% Make sure keyboard mapping is the same on all supported operating systems
+% Apple MacOS/X, MS-Windows and GNU/Linux:
+KbName('UnifyKeyNames');
+
+
+
+% ---------- FIX LATER ---------- %
```

Comment:
should we add this as an issue so that we don't forget?

Reply:
done